api/routes/api_routes.py

```diff
@@ -39,3 +40,23 @@ def update_report(id):
 @api.route('/api/v1/report/<id>', methods=['DELETE'])
 def destroy_report(id):
     return reportcontroller.destroy_report(id)
+
+@api.route('/api/v1/pmi/<id>', methods=['GET'])
```

Comment:
Great job with these routes. 
Question? Do we want to nest any of the models that are necessary for report under report. Like `/api/v1/report/pmi`?